### PR TITLE
Soarian fhir r4 document reference write

### PR DESCRIPTION
**Pull request by preethakarnan**
Description
----
Raising review request for Soarian FHIR R4 DocumentReference resource.
PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
DocumentReference_1
DocumentReference_2
DocumentReference_3
DocumentReference_4
DocumentReference_5

- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/soarian/r4/foundation/documents/document-reference.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource ID associated with the resource.
+ `patient`                | This, or `_id`     | [`reference`] | The patient who has the document referenced. 
+`category`                | No                 | [`token`]     | The categorization for the type of document referenced. Example: `category=clinical-note`
+`type`                    | No                 | [`token`]     | The kind of document referenced. Example: `https://loinc.org/11488-4`
+`date`                    | No                 | [`date`]      | Date range into which the document referenced falls. Example: `date=ge2020-01-01`
+`encounter`               | No                 | [`reference`] | The encounter associated with the document referenced. 
+`_count`                  | No                 | [`count`]     | The maximum number of resources returned in a page.
+`_revinclude`             | No                 | [`revinclude`]| A request to include any Provenance resource in the bundle that refers to a DocumentReference resource in the search results. Only supported with Provenance.
```
Comment (NeethuKP) [inline content/soarian/r4/foundation/documents/document-reference.md:64]: reinclude changes can be updated as discussed in the conversation
https://github.com/cerner/fhir.cerner.com/pull/753#discussion_r730742365

File: content/soarian/r4/foundation/documents/document-reference.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+### Parameters
+
+ Name                     | Required?          | Type          | Description
+--------------------------|--------------------|---------------|--------------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource ID associated with the resource.
+ `patient`                | This, or `_id`     | [`reference`] | The patient who has the document referenced. 
+`category`                | No                 | [`token`]     | The categorization for the type of document referenced. Example: `category=clinical-note`
+`type`                    | No                 | [`token`]     | The kind of document referenced. Example: `https://loinc.org/11488-4`
+`date`                    | No                 | [`date`]      | Date range into which the document referenced falls. Example: `date=ge2020-01-01`
```
Comment (NeethuKP) [inline content/soarian/r4/foundation/documents/document-reference.md:61]: should it be date_time ?

File: content/soarian/r4/foundation/documents/document-reference.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+* TOC
+{:toc}
+
+## Overview
+
+The DocumentReference resource is used to reference a clinical document for a patient in the health system. This resource supports reading clinical notes and returning references to retrieve Continuity of Care Documents (CCD).
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query. 
+
```
Comment (NeethuKP) [inline content/soarian/r4/foundation/documents/document-reference.md:15]: I think the API currently supports only 'GET' parameters

File: content/soarian/r4/foundation/documents/document-reference.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+### Authorization Types
+
+<%= authorization_types(provider: true, patient: true) %>
+
+### Parameters
+
+ Name                     | Required?          | Type          | Description
+--------------------------|--------------------|---------------|--------------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource ID associated with the resource.
+ `patient`                | This, or `_id`     | [`reference`] | The patient who has the document referenced.
```
Comment (HRNWriter) [inline content/soarian/r4/foundation/documents/document-reference.md:58]: I think this Description should be "The patient who the document references." I am verifying this with Stacey Schutawie (who owns this content) and Emily Keller.

PRODOC-160857,

File: content/soarian/r4/foundation/documents/document-reference.md (hunk 3 above)
Comment (preethakarnan) [reply content/soarian/r4/foundation/documents/document-reference.md:15]: we do support 'GET' and 'POST' both the methods.

File: content/soarian/r4/foundation/documents/document-reference.md (hunk 2 above)
Comment (preethakarnan) [reply content/soarian/r4/foundation/documents/document-reference.md:61]: We are keeping only date in order to make it consistent with the other resources.

**Review by NeethuKP — APPROVED** [review]
Ensure the comments are addressed before merging

**Review by slvsridhar — APPROVED** [review]

**Review by DeepakGupta93 — APPROVED** [review]

**Comment by slvsridhar** [comment]
After Publish : 
image
image
image
image
image

**Pull request merged**